web/content/release/katello-4.2.adoc

```diff
@@ -0,0 +1,68 @@
+Available guides:
+
+* link:/3.0/Planning_Guide/index-foreman-el.html[Planning for Foreman]
+* link:/3.0/Quickstart_Guide/index-foreman-el.html[Foreman Quickstart Guide on RHEL/CentOS]
+* link:/3.0/Quickstart_Guide/index-foreman-deb.html[Foreman Quickstart Guide on Debian/Ubuntu]
+* link:/3.0/Quickstart_Guide/index-katello.html[Katello Quickstart Guide on RHEL/CentOS]
+* link:/3.0/Installing_Server_on_Red_Hat/index-foreman-el.html[Installing Foreman on RHEL/CentOS]
+* link:/3.0/Installing_Server_on_Red_Hat/index-katello.html[Installing Katello on RHEL/CentOS]
+* link:/3.0/Installing_Proxy_on_Red_Hat/index-foreman-el.html[Installing Smart Proxy on RHEL/CentOS]
+* link:/3.0/Installing_Server_on_Debian/index-foreman-deb.html[Installing Foreman on Debian/Ubuntu]
+* link:/3.0/Installing_Proxy_on_Debian/index-foreman-deb.html[Installing Smart Proxy on Debian/Ubuntu]
+* link:/3.0/Upgrading_and_Updating/index-foreman-el.html[Upgrading and Updating Foreman]
+* link:/3.0/Upgrading_and_Updating/index-katello.html[Upgrading and Updating Katello]
+* link:/3.0/Deploying_on_AWS/index-foreman-el.html[Deploying Foreman on AWS (reference guide)]
+* link:/3.0/Configuring_Load_Balancer/index-foreman-el.html[Configuring Smart Proxies with a Load Balancer]
+* link:/3.0/Provisioning_Guide/index-foreman-el.html[Provisioning Guide]
+* link:/3.0/Content_Management_Guide/index-katello.html[Content Management Guide]
+* link:/3.0/Configuring_Ansible/index-foreman-el.html[Configuring Foreman to use Ansible]
+* link:/3.0/Managing_Hosts/index-foreman-el.html[Managing Hosts Guide]
+* link:/3.0/Administering_Red_Hat_Satellite/index-foreman-el.html[Administering Foreman]
+* link:/3.0/Application_Centric_Deployment/index-foreman-el.html[Application Centric Deployment]
+
+== Headline features
+
+For the full list of changes, see the
+https://github.com/Katello/katello/blob/KATELLO-4.2/CHANGELOG.md[Changelog].
+
+* New Experimental Content View UI Features:
+** Publish & Promote
+** Add component CVs to composite CVs
+* Pulp’s mirorred metadata is now used to speed up repo syncs
+* Upgrade to Pulp 3.14 (This was backported to Katello 4.1)
+* support deleting RH repos from the repositories details
+
+== Upgrade Warnings
+
+* upstream_subscriptions apis have been removed in favor of apis rooted
+at /katello/api/v2/organizations/:organization_id/simple_content_access/
+
+== Deprecations
+
+== Contributors
+
+* Andrew Dewar
+* Andrew Lukoshko
+* Bernhard Suttner
+* Chris Roberts
+* Dominik Matoulek
+* Eric D. Helms
+* Hao Yu
+* Ian Ballou
+* James Jeffers
+* Jeremy Lenz
+* Jonathon Turel
+* Justin Sherrill
+* Leos Stejskal
+* Lucy Fu
+* Lukas Zapletal
+* lumarel
+* Marek Hulán
+* Markus Bucher
+* Oleh Fedorenko
+* Ondřej Ezr
+* Partha Aji
+* Pat Riehecky
+* Paul Donohue
+* Ryan Verdile
+* Samir Jha
```

Comment:
We just merged https://github.com/theforeman/foreman-documentation/pull/698 and https://github.com/theforeman/foreman-documentation/pull/699. Can we link to that guide instead?

Reply:
Sure, as long as it's published, I don't see it yet: https://docs.theforeman.org/Release_notes/ or https://docs.theforeman.org/3.0/Release_notes/